src/route/api-address-name-service.js

```diff
@@ -0,0 +1,16 @@
+const { Router } = require('express');
+const { onRequestSuccess, onRequestError } = require('./utils');
+const AddressNameServiceApi = require('../api/address-name-service');
+
+const router = Router();
+
+router.get('/resolveAddress', (req, res, next) => {
```

Comment:
can delete this, we dont need to expose this API to the public. the UI shouldn't call this ever and all the names should be returned in all the graph queries.